packages/commands/src/components/command-menu.js

```diff
@@ -287,7 +277,10 @@ export function CommandMenu() {
 						/>
 						<Icon icon={ inputIcon } />
 					</div>
-					<Command.List ref={ commandListRef }>
+					<Command.List
+						ref={ commandListRef }
```

Comment:
I think this ref can be removed now?

Reply:
Nice catch! Indeed, the ref is no longer necessary. Deleted by [0a0ecc9](https://github.com/WordPress/gutenberg/pull/63465/commits/0a0ecc97ce0b14c1f1a7e536a36659cfcf5e99eb)